plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/Library.java

```diff
@@ -173,44 +171,53 @@ void setTransport(String transport) {
     this.transport = transport;
   }
   
-  public synchronized boolean isResolved() {
-    return this.resolved;
+  @VisibleForTesting
+  synchronized boolean isResolved() {
+    return transitiveDependencies != null;
   }
   
   /**
    * @param resolved true iff this library contains its complete dependency graph
    */
   synchronized void setResolved(boolean resolved) {
```

Comment:
No one calls the method with the param value of `false`, so I think we can remove the parameter too. Maybe the method name can be changed after that.

Reply:
done